### DISCUSSION TITLE
"Looks like normal points into the shape"

### DISCUSSION THREAD

**Discussion by bhushan6** [pmndrs/use-cannon#245] [Q&A] [answered]
I am trying use convex polyhedron, but its giving an error saying that normal is pointing inside the shape, but when i checked the nomals of my glb model they are all pointing out in right direction, is it because I am importing model from blender and blender and three has different axis directions? How can I solve this issue?

**Comment by stockhuman** [comment, marked as answer]
It's possible to get this error with a non-convex shape. What's the model?

**Reply by bhushan6** [reply]
Yes, that was an issue. Thanks :)
